ISA-XLSX.md

```diff
@@ -709,6 +749,14 @@ A `Parameter` can be used to specify any additional information about the experi
 |--------------------------------|--------|-------------------|------------------------------------------------------|
 |                            300 | Kelvin | UO                | [http://…/obo/UO_0000032](http://purl.obolibrary.org/obo/UO_0000032) |
 
+## Comments
+
+A `Comment` can be used to provide some additional information. Columns headed with `Comment[<comment name>]` MAY appear anywhere in the Annotation Table. The comment always refers to the Annotation Table. The value MUST be free text.
+
+| Comment [Answer to everything] |
+|--------------------------------|
+|                      forty-two |
+
 ## Others
 
 Columns whose headers do not follow any of the formats described above are considered additional payload and are out of the scope of this specification.
```

Comment:
Does this mean we now officially support free text columns?

Reply:
I'd read that as "the tool implementing the standard is free to decide what to do with free text columns"

Reply:
Yes, just a heads-up that a hard fail on unknown columns is not necessary.